src/main/scala/com/fulcrumgenomics/vcf/AssessPhasing.scala

```diff
@@ -56,17 +56,23 @@ import scala.collection.mutable.ListBuffer
       |The input VCFs are assumed to be single sample: the genotype from the first sample is used.
       |
       |Only bi-allelic heterozygous SNPs are considered.
+      |
+      |The input known phased variants can be subsetted using the known interval list, for example to keep only variants
+      |from high-confidence regions.
+      |
+      |If an interval list is given, only the set of chromosomes specified will be analyzed.  Note that the full
+      |chromosome will be analyzed and start/stop positions will be ignored.
     """,
   group=ClpGroups.VcfOrBcf
 )
 class AssessPhasing
 ( @arg(flag="c", doc="The VCF with called phased variants.") val calledVcf: PathToVcf,
   @arg(flag="t", doc="The VCF with known phased variants.") val truthVcf: PathToVcf,
   @arg(flag="o", doc="The output prefix for all output files.") val output: PathPrefix,
-  @arg(flag="l", doc="The interval list over which known phased variants should be kept.") val knownIntervals: Option[PathToIntervals] = None,
+  @arg(flag="k", doc="The interval list over which known phased variants should be kept.") val knownIntervals: Option[PathToIntervals] = None,
   @arg(flag="m", doc="Allow missing fields in the VCF header.") val allowMissingFieldsInVcfHeader: Boolean = true,
   @arg(flag="s", doc="Skip sites where the truth and call are both called but do not share the same alleles.") val skipMismatchingAlleles: Boolean = true,
-  @arg(flag="C", doc="Analyze only the given chromosome.") val chromosome: Option[String] = None,
```

Comment:
This still seems like it would be useful on occasion no?  Why not keep it and make it mutex with `intervals` and then if specific, use it go generate an `IntervalList`?  Up to you though since I don't really ever use this tool.